php/basicphp/config.yaml

```diff
@@ -1,5 +1,5 @@
 framework:
   github: ray-ang/basicphp
-  version: 0.9
+  version: 0.9.9
```

Comment:
please avoid patch level

Reply:
Thanks. I reverted the line back to version level -> 0.9